syntax = "proto3";

service AutoDataCollection {
    // 客户端发起请求，服务端返回 xml 字符串
    rpc txt2xml(auth) returns (resultXML) {}
}

message auth {
    string username = 1;
    string password = 2;
}

message resultXML {
    string result = 1;
}